test/nonregression/pipelines/test_run_pipelines_dl.py

```diff
@@ -44,25 +44,32 @@ def test_run_DLPrepareData(cmdopt):
     roi_params = {
         "extract_method": "roi",
         "roi_list": ["rightHippocampusBox", "leftHippocampusBox"],
-        "roi_uncrop_output": False,
+        # "roi_uncrop_output": False,
```

Comment:
These two parameter are not exchangeable: `roi_uncrop_output` allows to have cropped versions of the outputs (around the roi) while `use_uncropped_image` indicates if the input images are cropped or not.

Reply:
I am glad I asked

Reply:
I know they are not exchangeable and I did aim at using the later as per the data for the non-regression. Is there still an issue?

Reply:
I had the impression that you were exchanging these parameters... anyway you need to verify if the test reference data uses the cropped version or not. If yes, this line must be uncommented.

Reply:
No sorry for the confusion, these were two different changes.
`roi_uncrop_output` was commented so that the output is coherent with the data in the ref folder
`use_uncropped_image` was added because of the masks we have as input are uncropped.

I tested with the parameters locally on the reference data we had and the tests passed on my machine. Unless the data in the CI machines have changed, it should be ok.